test/vault_helpers.go

```diff
@@ -379,29 +255,7 @@ func runVaultEnterpriseClusterTest(t *testing.T, packerBuildName string, sshUser
 	})
 
 	test_structure.RunTestStage(t, "deploy", func() {
-		uniqueId := random.UniqueId()
-		amiId := test_structure.LoadAmiId(t, examplesDir)
-		awsRegion := test_structure.LoadString(t, examplesDir, SAVED_AWS_REGION)
-
-		keyPair := aws.CreateAndImportEC2KeyPair(t, awsRegion, uniqueId)
-		test_structure.SaveEc2KeyPair(t, examplesDir, keyPair)
-
-		terraformOptions := &terraform.Options{
-			TerraformDir: examplesDir,
-			Vars: map[string]interface{}{
-				VAR_AMI_ID:                 amiId,
-				VAR_VAULT_CLUSTER_NAME:     fmt.Sprintf("vault-test-%s", uniqueId),
-				VAR_CONSUL_CLUSTER_NAME:    fmt.Sprintf("consul-test-%s", uniqueId),
-				VAR_CONSUL_CLUSTER_TAG_KEY: fmt.Sprintf("consul-test-%s", uniqueId),
-				VAR_SSH_KEY_NAME:           keyPair.Name,
-			},
-			EnvVars: map[string]string{
-				ENV_VAR_AWS_REGION: awsRegion,
-			},
-		}
-		test_structure.SaveTerraformOptions(t, examplesDir, terraformOptions)
-
-		terraform.InitAndApply(t, terraformOptions)
+		deployCluster(t, examplesDir, random.UniqueId(), map[string]interface{}{})
```

Comment:
Can you replace ` map[string]interface{}{}` with just `nil`?

Reply:
Sure!